pkg/executables/tink.go

```diff
@@ -57,7 +57,9 @@ func (t *Tink) GetHardware(ctx context.Context) ([]*hardware.Hardware, error) {
 			return nil, fmt.Errorf("error unmarshling hardware json: %v", err)
 		}
 		if len(hardwareListData["data"]) > 0 {
-			hardwareList = append(hardwareList, hardwareListData["data"]...)
+			for _, data := range hardwareListData["data"] {
+				hardwareList[data.GetId()] = data
```

Comment:
I think we can leave this function as it was and form the `uuid: hardware` map while doing validations.
This method was intended to make the `tink get hardware` call and return a list of hardwares.